src/styles/blocks/header.scss

```diff
@@ -0,0 +1,141 @@
+.header {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+
+  background-image: url(../images/Header.png);
+  background-repeat: no-repeat;
+  background-position: top;
+  background-size: cover;
+
+  &__content-wrapper {
+    padding: 36px 10%;
```

Comment:
why 10%?

Reply:
changed to 120px - was an attempt to keep it reactive.